### PR TITLE
 Adding SMS Feedback Details

### PR DESCRIPTION
**Pull request by nikshar**
Please link to the issue #…
https://github.com/adobe/xdm/issues/1448

### CODE REVIEW

**Comment by nikshar** [comment]
Hi @bisin-adobe ,
Can you take a look to this PR.
this is needed for AJO and the Issue id is : https://github.com/adobe/xdm/issues/1448

File: extensions/adobe/experience/customerJourneyManagement/message-delivery-feedback.schema.json [hunk 1]
```diff
@@ -37,7 +38,8 @@
             "bounce": "Message bounced",
             "delay": "Message delayed",
             "duplicate": "Duplicate message skipped",
-            "denylist": "Invalid destination skipped"
+            "denylist": "Invalid destination skipped",
+            "delivered": "Message Delivered"
```
Comment (sawhneyh) [inline extensions/adobe/experience/customerJourneyManagement/message-delivery-feedback.schema.json:42]: As **delivered** is not explicitly passed with other channels, we should update the description for the feedbackStatus that some enum values may not be applicable for all channels.

Comment (nikshar) [reply extensions/adobe/experience/customerJourneyManagement/message-delivery-feedback.schema.json:42]: updated

**Comment by nikshar** [comment]
@prabhum2 : Can we review this one

File: extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json [hunk 2]
```diff
@@ -79,6 +79,11 @@
           },
           "meta:titleId": "messageprofile##https://ns.adobe.com/experience/customerJourneyManagement/pushChannelContext/platform##title##73351",
           "meta:descriptionId": "messageprofile##https://ns.adobe.com/experience/customerJourneyManagement/pushChannelContext/platform##description##33451"
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/smsChannelContext/provider": {
+          "title": "SMS Provider",
+          "type": "string",
+          "description": "SMS provider , e.g. sinch or twilio"
```
Comment (sawhneyh) [inline extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json:86]: If this is a pre defined list then we can make this enum.

**Review by sawhneyh — APPROVED** [review]

File: extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json (hunk 2 above)
Comment (nikshar) [reply extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json:86]: done

**Review by sawhneyh — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**